data/currency/version_1.0.0/Description.md

```diff
@@ -0,0 +1,8 @@
+| Title         | ---                                                                                                                                                                                                                                                                                                                                                                                                            |
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Do we need to maintain versioning in diff folders ? 
Can we not establish this by tags in github ?

It will really impact the size of repo, if we keep adding new data in different directories.